package.json

```diff
@@ -26,16 +26,17 @@
   "keywords": [
     "react-native",
     "expo",
-    "@jellyfish-dev/react-native-client-sdk",
-    "MembraneWebRTC"
+    "@fishjam-dev/react-native-client-sdk",
+    "MembraneWebRTC",
+    "Fishjam"
   ],
-  "repository": "https://github.com/membraneframework/react-native-client-sdk",
+  "repository": "https://github.com/fishjam-dev/react-native-client-sdk",
   "author": "Angelika Serwa <angelika.serwa@swmansion.com> (https://github.com/graszka22)",
```

Comment:
Know that it is not your code but lets update author to software mansion.

Reply:
I was planning to do that before releasing new version of lib. But updated it here